mlir/test/Dialect/MQTDyn/IR/dialect_features.mlir

```diff
@@ -571,6 +571,8 @@ module {
     func.func @testAllocMissingSize() {
         // expected-error-re@+1 {{'mqtdyn.allocQubitRegister' op exactly one attribute ({{.*}}) or operand ({{.*}}) must be provided for 'size'}}
         %qreg = "mqtdyn.allocQubitRegister"() : () -> !mqtdyn.QubitRegister
+
+        return
```

Comment:
Is that return not implicit? Or, asked differently, does it really have to be explicit?
I suspect it is good practice to keep it; if only for consistency.

Reply:
Ah. That makes sense! Thanks for the explanation.